bazel/cc_toolchains/clang_configuration.bzl

```
@@ -100,24 +104,26 @@ def _compute_bsd_sysroot(repository_ctx):
         return sysroot_path.realpath
     return default
 
+# File content used when computing search paths. This additionally verifies that
+# libc++ is installed.
+_CLANG_INCLUDE_FILE_CONTENT = """
+#if __has_include(<utility>)
+#include <utility>
```

As suggested by @zygoloid -- can probably even more cleanly use:

```suggestion
#if __has_include(<version>)
#include <version>
```